app/controllers/api/v1/campaign_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::CampaignController < ApiController
+  def index
+    @page = params['page'] ? params['page'] : 1
+    @sort_by = params['sort_by'] ? params['sort_by'] : 'position'
+    @direction = params['direction'] ? params['direction'] : 'ASC'
+    @campaigns = Campaign.paginate(:page => @page, :per_page => 3).order(@sort_by + ' ' + @direction)
+
+    render json: @campaigns
```

Comment:
similar to my suggestion on the events endpoint I think we should use the ActiveModelSerializer.